test/rule-matches/not-html-matches.js

```diff
@@ -1,20 +0,0 @@
-describe('not-html-matches', function() {
-  'use strict';
-
-  var fixture = document.getElementById('fixture');
-  var rule;
-
-  beforeEach(function() {
-    rule = axe.utils.getRule('valid-lang');
```

Comment:
There's no way to run these tests, since there's no way to get at the matcher. I'm deprecating the matcher in #3524